DECISION-MAKING.md

```diff
@@ -161,22 +135,15 @@ Repository:
 
 ## IAM Infrastructure
 
-- Owner: @adelbarrio
-- Peers: @ziegeer @fiji-flo
-- Operator: EIS, SREs
 
 ## OpenLDAP
 
 This is the Staff user database, credential storage and group system (IdP). It also contains some non-staff contributors.
 
-- Owner: @jdow
+- Owner: @rtucker
 - Peers: @gcox
 - Operator: CoreServices
 
 ## DuoSecurity (2FA)
 
 This is the 2FA/MFA product utilized to augment OpenLDAP with a second-factor.
-
-- Owner: @gene1wood
-- Peers: @april @tristanweir
-- Operator: Third party (Duo)
```

Comment:
I think we should keep this Operator line.

Reply:
Sure, thing, added it back in.